### PR TITLE
Scrolling Object Collection - Adding new example scene

### PR DESCRIPTION
**Pull request by vaoliva**
## Overview
Adding a draft example scene for Scrolling Object Collection.

Adding @cre8ivepark to the branch for further polish

**App :** [HL2 Build](https://microsoft-my.sharepoint.com/:u:/p/vaolivaa/EfzcMNq-7OBDvzKO4vTSqKYBE1cFvCrdEB3CoCdqQ0zsqw?e=WzR9q5)

scrolling_graduation

### CODE REVIEW

**Comment by julenka** [comment]
This is much improved compared to the previous experimental state of scroll object collections! Couple of feedbacks after trying the appx:

1) The overscroll is not have any damping applied to it. Try scrolling on the shell pins panel you'll see that the farther you move the more resistance the scroll has.

2) Scroll via touch: When the finger leaves the touchable area I hear a touch up sound even though I'm still engaged / touching the scroll view. When I enter the scroll area again, I hear a sound.

3) Far scroll - when the button gets unpressed due to a scroll, I hear a button up sound. I would not expect to hear a button up sound if a button gets unpressed because I am scrolling, only when it's unpressed due to a click.

4) Touch scrolling -- the scroll amount is not 1:1. When I move my finger down 5 cm, the resulting scroll content moves quite a bit more than 5 cm, as if the scroll is amplified. Try scrolling on your phone -- notice that when you touch the phone and scroll, if you move your finger 1 cm, content moves 1 cm. If you try this on the start menu in shell you'll see the same thing. I expect the scroll amount to be 1:1

**Comment by julenka** [comment]
minor: the lunar lander in "scroll view no pagination" is upside down

File: Assets/MRTK/Examples/Experimental/ScrollingObjectCollection/Materials/MRTK_Standard_Violet.mat [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    - _IndependentCorners: 0
+    - _InnerGlow: 0
+    - _InnerGlowPower: 4
+    - _InstancedColor: 0
+    - _Iridescence: 0
+    - _IridescenceAngle: -0.78
+    - _IridescenceIntensity: 0.5
+    - _IridescenceThreshold: 0.05
+    - _Lightmapping: 1
+    - _Metallic: 0
```
Comment (julenka) [inline Assets/MRTK/Examples/Experimental/ScrollingObjectCollection/Materials/MRTK_Standard_Violet.mat:131]: Should we move these materials into the mrtk standard materials, or re-use existing ones?

**Review by julenka — COMMENTED** [review]
Made a couple comments regarding experience bugs that should be addressed.

File: Assets/MRTK/Examples/Experimental/ScrollingObjectCollection/Materials/MRTK_Standard_Violet.mat (hunk 1 above)
Comment (vaoliva) [reply Assets/MRTK/Examples/Experimental/ScrollingObjectCollection/Materials/MRTK_Standard_Violet.mat:131]: I will check this with @cre8ivepark as these are not duplications. They are new colors. But if needed we can add a script for changing colors of instanced materiasls.

**Comment by vaoliva** [comment]
> The overscroll is not have any damping applied to it. Try scrolling on the shell pins panel you'll see that the farther you move the more resistance the scroll has.

Scroll via touch: When the finger leaves the touchable area I hear a touch up sound even though I'm still engaged / touching the scroll view. When I enter the scroll area again, I hear a sound.

Far scroll - when the button gets unpressed due to a scroll, I hear a button up sound. I would not expect to hear a button up sound if a button gets unpressed because I am scrolling, only when it's unpressed due to a click.

Touch scrolling -- the scroll amount is not 1:1. When I move my finger down 5 cm, the resulting scroll content moves quite a bit more than 5 cm, as if the scroll is amplified. Try scrolling on your phone -- notice that when you touch the phone and scroll, if you move your finger 1 cm, content moves 1 cm. If you try this on the start menu in shell you'll see the same thing. I expect the scroll amount to be 1:1

Will address all these for stabilization.

**Comment by cre8ivepark** [comment]
Pushed some design polish items:
- Updated with new description panel
- Updated scroll view examples in the center and right: color, layout, card design polish.

2020-09-04 23_50_16-Unity 2018 4 26f1 Personal -  PREVIEW PACKAGES IN USE  - ScrollingObjectCollecti

2020-09-04 23_49_59-Unity 2018 4 26f1 Personal -  PREVIEW PACKAGES IN USE  - ScrollingObjectCollecti

**Review by julenka — APPROVED** [review]
I tested the experience in device and it felt quite good, no new issues. Great job!

**Review by cre8ivepark — APPROVED** [review]
Tested on the device and it looks great!

**Pull request merged**